app/src/main/java/org/mozilla/tv/firefox/components/locale/LocaleManager.java

```diff
@@ -288,7 +289,16 @@ public Boolean currentLanguageIsEnglish(@NonNull Context context) {
         Locale current = getCurrentLocale(context);
         // If locale hasn't been updated (i.e., 'current' is null), use system default
         if (current == null) {
-            current = context.getResources().getConfiguration().locale;
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                current = context.getResources().getConfiguration().getLocales().get(0);
+            } else {
+                current = context.getResources().getConfiguration().locale;
+            }
+        }
+        // In a very small number of cases, this locale will still be null. Most of our
+        // userbase uses English as a primary language, so we default to that as a fallback
+        if (current == null) {
```

Comment:
Can we add a Sentry log here? I'm genuinely curious to see if how many reach here after adding deprecation check